zipkin-server/README.md

```diff
@@ -459,6 +477,19 @@ $ COLLECTOR_GRPC_ENABLED=true java -jar zipkin.jar
 
 As this service is experimental, it is not recommended to run this in production environments.
 
+## Self-Tracing
+Self tracing exists to help troubleshoot performance of the zipkin-server. Production deployments
+who enable self-tracing should lower the sample rate from 1.0 (100%) to a much smaller rate, like
+0.001 (0.1% or 1 out of 1000).
+
+When `zipkin.self-tracing.enabled=true`, Zipkin will self-trace calls to the api.
```

Comment:
How about specifying what is the name of the service or how to find such traces?

Reply:
done!